src/lib/issue.ts

```diff
@@ -70,29 +67,17 @@ export class Issue {
     this.author = issue.author;
     this.title = issue.title;
     this.state = issue.state;
-    this.comment = issue.comment; // TODO: Remove this after we have migrated to Heartwood.
     this.discussion = issue.discussion;
     this.tags = issue.tags;
     this.assignees = issue.assignees;
-    if (window.HEARTWOOD) {
-      this.timestamp = issue.discussion[0].timestamp;
-    } else {
-      this.timestamp = issue.timestamp;
-    }
+    this.timestamp = issue.discussion[0].timestamp;
   }
 
   // Counts the amount of comments and replies in a discussion
   countComments(): number {
-    if (window.HEARTWOOD) {
-      return this.discussion.reduce(acc => {
-        return acc + 1; // If there are no replies, we simply add 1 for the comment in this loop.
-      }, 0);
-    } else {
-      return this.discussion.reduce((acc, comment) => {
-        if (comment.replies) return acc + comment.replies.length + 1; // We add all replies and 1 forathe comment in this loop.
-        return acc + 1; // If there are no replies, we simply add 1 for the comment in this loop.
-      }, 0);
-    }
+    return this.discussion.reduce(acc => {
+      return acc + 1; // If there are no replies, we simply add 1 for the comment in this loop.
```

Comment:
:thinking: At some point we added the `+ 1` to not have issues with 0 comments. But since we now always will have the initial description as part of the `discussion` thread, maybe we need to skip the first item or don't count the first comment..

Reply:
Hm, I think it's kind of weird that we separate the issue title from the body. Now the body is a comment. If we remove the +1 then we lose the "1 comment" icon in the listing and the issue shows one comment which is the body.

<img width="551" alt="Screenshot 2023-03-03 at 09 29 35" src="https://user-images.githubusercontent.com/158411/222670800-1dfa9fd4-07f7-4e83-b2ee-6840225d54ba.png">
<img width="856" alt="Screenshot 2023-03-03 at 09 29 41" src="https://user-images.githubusercontent.com/158411/222670810-ab9f8d9f-a065-460e-bd6d-173e26dfdc62.png">

If we keep the body separate as a comment, I think it's more consistent to leave the +1.

Reply:
Yeah I'm also not sure on having the description as a first comment tbh. It sounds better to me to have it together with the issue title, but for consistency we can leave it with the +1